src/catalyst-toolbox/catalyst-toolbox/src/proposal_score/store.rs

```diff
@@ -1,75 +1,97 @@
-use super::{AligmentScore, AuditabilityScore, FeasibilityScore, ProposalId};
-use rusqlite::Connection;
-use std::{path::PathBuf, str::FromStr};
+use super::{AligmentScore, AuditabilityScore, FeasibilityScore};
```

Comment:
@Mr-Leshiy typos in `Aligment***` types, should be `Alignment***`

Reply:
done